### ISSUE TITLE
Visual Studio SSDT and Access Database Engine Connections in Tabular Model

### ISSUE THREAD

**Issue by CubeSpark** [MicrosoftDocs/powerbi-docs#2650]
Since Visual Studio SSDT is a 32-Bit COM application, you MUST install the 32-bit OLEDB provider for Access (the so-called ACE Driver) in order to be able to connect to a 64-bit ACCDB file in ANY CASE.  I downloaded the Access 32 Bit Redistributable RunTime, and I was able to connect through a UNC Pathname to my sandbox developer's ACCDB database.  Get it from here, and select the non-64 bit version:  https://www.microsoft.com/en-us/download/details.aspx?id=13255.

Step 1:  NET SHARE your Access ACCDB file.    I used C:\NET SHARE Muscles = "\\Calayugan\Directory\"
Step 2. Install the Redistributable from https://www.microsoft.com/en-us/download/details.aspx?id=13255.
Step 3. Install it from the command prompt elevated to Administrator, and use the "/passive" switch
Step 4:  Go to Visual Studio 2017 / 2019, and create a new Tabular Model
Step 5: Create the Data Source
Step 6: Point through the share to the thing (ACCDB)
Step 7: Impersonate the Joker if that is your Administrative Login

Play!

Don't forget to check my YouTube Channel:   "Calayugan Music Man" for really BORING videos on exciting topics

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 680485c7-5823-2c7f-df5d-b8c8a8205759
* Version Independent ID: 5a217642-7723-6980-28ae-e26559c24201
* Content: [Resolve Access and .XLS import issues in Power BI Desktop - Power BI](https://docs.microsoft.com/en-us/power-bi/connect-data/desktop-access-database-errors)
* Content Source: [powerbi-docs/connect-data/desktop-access-database-errors.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/master/powerbi-docs/connect-data/desktop-access-database-errors.md)
* Service: **powerbi**
* Sub-service: **powerbi-desktop**
* GitHub Login: @davidiseminger
* Microsoft Alias: **davidi**

**Comment by mihart** [comment]
Thanks for your comment, @CubeSpark . This issue has been assigned to the doc author for review.

**Comment by mihart** [comment]
Hi @CubeSpark , 
Thank you for your suggestion.  I chatted with the Product team and they believe that the 32 vs 64 mismatch is covered in the existing document and a separate section for SSDT is not necessary.   Here is their response:
"IMO, this is already captured in the sense that the article already covers 32-bit vs. 64-bit. SSDT is an example in this case of 32-bit installation, but same would be true for Excel or Power BI Desktop.".  I did add a sentence  that the solutions in the article can be applied to other mismatches.  Take a look, and let me know if you believe that the article still needs an update. The new version should go live by end of the week. 
All the best,
The DOCS team
